javascript/demo/demo.html

```diff
@@ -1,24 +1,22 @@
 <!doctype html>
 <html>
   <script>
-    var Module = {
-      onRuntimeInitialized: function() {
-          fetch("http://host:port/result/<pid>/stream", {
-              headers: {
-                  'Authorization': 'Bearer <token>'
+    CreateOneSeismic().then( oneSeismic => {
+        fetch("http://host:port/result/<pid>/stream", {
+            headers: {
+                'Authorization': 'Bearer <token>'
               }
-         })
+        })
         .then(response => response.arrayBuffer())
         .then(function (chunk) {
               console.log(chunk);
-              let dec = Module.stream_decoder();
+              let dec = oneSeismic.stream_decoder();
               var x = dec(chunk);
               x = dec(null);
               const a = x[1]['data']['data'];
               console.log(a.reduce((acc, v) => acc +v ));
         })
-      }
-    };
+    });
   </script>
   <script src="oneseismic.js"></script>
```

Comment:
Yes because no longer using the onRuntimeInitialized hook

Reply:
Yep, you can move it in new html thing too.
And now the code is reverted to an old version, when it would be nice to have new version of demo but with just working CreateOneseismic thingy.
I am still checking if they work :D